src/components/instructions.sass

```diff
@@ -0,0 +1,6 @@
+@import vars
+
+.authored-markdown
+  font-family: $font-stack
+  padding: $component-padding
+
```

Comment:
@sfentress This sass file is sparse for now, but I wanted it separated because I assume we will have to add more formatting to the authored block once PIs have some time to play with the feature.